### ISSUE TITLE
[Screen Reader-Help-Projects]: Narrator focus not in proper order inside Projects screen in Scan mode.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2568]
**User Experience:**
Users who depends on Screen reader will get confuse if Narrator focus not in proper order inside Projects screen in Scan mode.

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.

**Pre-Requisite:** Turn on Narrator
Scan mode : (Caps + Space to set Narrator to scan mode . Use Up/Down arrow keys to navigate )

**Repro Steps:-**
1- Navigate to https://makecode.microbit.org/acc
2- Navigate to the Microbit section element and select Code control given on it.
3- Navigate to the Help control lying in the header section on the page and select it.
4- Navigate to 'Projects' control and select it.
5- Navigate to various controls lying on the pane opened.
6- Verify that Screen Reader behavior.

**Actual Result:-**
In Scan mode(CAPS + Space), Narrator focus not in proper order inside Projects screen as it moves like 'Open a documentation in New tab' link -->Projects header

**Expected Result:-**
In Scan mode(CAPS + Space), Narrator focus should be in  proper order inside Projects Buttons screen as it should move  like Projects header--> 'Open a documentation in New tab' link

**Note:**
This screen is not supported by NVDA properly
.
Same issue exists in Narrator items mode(CAPS +Arrow keys) as well. 

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/Documents/Forms/AllItems.aspx?id=%2Fteams%2Fmsenable%2FDocuments%2FLCA%2FMAS%20%2D%20July%201%2C%202017%20Release%2FMAS%204%2E2%2E10%20%E2%80%93%20Tracking%20of%20Focus%20and%20Selection%20Attributes%2Edocx&parent=%2Fteams%2Fmsenable%2FDocuments%2FLCA%2FMAS%20%2D%20July%201%2C%202017%20Release

Please refer the attachment
[MAS4.2.10_Projects_Narrator focus not in order.zip](https://github.com/Microsoft/pxt/files/1193659/MAS4.2.10_Projects_Narrator.focus.not.in.order.zip)

**Comment by v-mesala** [comment]
Verified on URL:  https://makecode.microbit.org/acc and issue is fixed. Hence closing the bug.
